lib/thread1.g

```diff
@@ -6,15 +6,86 @@
 #Y  (C) 1998 School Math and Comp. Sci., University of St.  Andrews, Scotland
 #Y  Copyright (C) 2002 The GAP Group
 ##
-##  This file provides trivial mocks of thread-related primitives for 
+##  This file provides trivial mocks of thread-related primitives for
 ##  traditional GAP.
 ##
 
+# First we define some objects defined in init.g in HPC-GAP
+FILTER_REGION	 := "filter region";
+ThreadVar := rec();
+
+# Now we mock functions from thread1.g
+
+MakeThreadLocal := function(l) return l; end;
+MakeReadOnly := function(l) return l; end;
+MakeReadOnlyRaw := function(l) return l; end;
+MakeProtected := function(l) return l; end;
+MakeProjectedObj := function(l) return l; end;
+MakeReadOnlyObj := function(l) return l; end;
+
```

Comment:
In HPC-GAP, these and others all are read-only, e.g.:
```
IsReadOnlyGVar("MakeThreadLocal");
```
So perhaps this should be using `BindGlobal`. I'd also consider using `IdFunc` for the value, as (a) some code optimizes for that, and (b) it saves a tiny amount of memory ;-). So we'd write:
```
BindGlobal("MakeThreadLocal", IdFunc);
```